src/screens/MainScreen/index.js

```diff
@@ -27,8 +28,16 @@ const MainScreen = () => {
           latitudeDelta: LATITUDE_DELTA,
           longitudeDelta: LONGITUDE_DELTA,
         }}
-        showUserLocation
-      />
+        showUserLocation>
+        {targets.map(element => (
+          <CustomMarker
+            key={element.target.id}
+            lat={element.target.lat}
+            lng={element.target.lng}
+            topicId={element.target.topicId}
+          />
+        ))}
```

Comment:
Instead of using element you can destruct and have `({target: {id, lat, lng, topicId}})` and avoid writing element.target every time